Traceback (most recent call last):
  File "/home/mukhtar/misogi/Medical-AI-Assistant/main.py", line 4, in <module>
    from fastapi import FastAPI, HTTPException
ModuleNotFoundError: No module named 'fastapi'
